From 70067d2092cd3a5e5465d8667dad274b479ea05c Mon Sep 17 00:00:00 2001
From: "Andre, Gerard" <gerard.andre@intel.com>
Date: Thu, 7 Apr 2016 10:11:33 +0100
Subject: [PATCH 5/5] Revert "Touch : add management of GPIO1 regulator."

This patch breaks the SD card detection.

This reverts commit 04adde617fbda8a34c75cb88b3c8013e44e4cc14.
This reverts patch https://android.intel.com/#/c/483018

Change-Id: Idcd830eb2d23b6f356b1449ed65433517a0beb93
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-26918
Reviewed-on: https://android.intel.com:443/491751
---
 drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c b/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
index ee3ffa2..245fc4e4 100644
--- a/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
+++ b/drivers/input/touchscreen/synaptics_dsx/synaptics_dsx_rmi_hid_i2c.c
@@ -147,7 +147,6 @@ static struct synaptics_dsx_board_data s7300_board_data = {
 	.cap_button_map = &cap_button_map,
 	.vir_button_map = &vir_button_map,
 	.device_descriptor_addr = DSX_HID_DEVICE_DESCRIPTOR_ADDR,
-	.pwr_reg_name = "GPIO1",
 };
 
 static int do_i2c_transfer(struct i2c_client *client, struct i2c_msg *msg)
-- 
1.9.1

